plugins/custom/docker_custom.py

```diff
@@ -181,13 +181,23 @@ def _read_task_stats(self):
                                  humanize_bytes(net_r),
                                  humanize_bytes(net_w)))
 
-            if cpu_percent < 5:
-                unhealthy_count += 1
-            else:
-                unhealthy_count = 0
+            try:
+                 info = self.cli.inspect_container(container=self.container['Id'])
+                 health_check_ok = info['State']['Health']['Status']
+                 if health_check_ok:
```

Comment:
Is there any content in the health check status? Or does every value other than `None`/`False` mean that things are ok?

Reply:
When I tested it, when there is a health check for the container, `info['State']['Health']['Status']` is either `True` or `False`,  The `Health` key does not exist if the image does not have a health check and the container is not started with user specified health check.

Reply:
Got it. Sounds reasonable.